include/sfx.h

```diff
@@ -393,15 +392,16 @@ enum Sfx {
     SFX_STONE_MOVE_A,
     SFX_STONE_MOVE_B,
     SFX_STONE_MOVE_C,
-    SFX_SWORD_SWISH_A,
-    SFX_SWORD_SWISH_B,
-    SFX_SWORD_SWISH_C,
-    SFX_SKULL_HIT_A,
-    SFX_SKULL_HIT_B,
-    SFX_SKULL_HIT_C,
-    SFX_SKULL_HIT_D,
-    SFX_SKULL_HIT_E,
-    SFX_SKULL_HIT_F,
+    SFX_WEAPON_SWISH_A,
+    SFX_WEAPON_SWISH_B,
+    SFX_WEAPON_SWISH_C,
+    SFX_METAL_CLANG_A,
+    SFX_METAL_CLANG_B,
+    SFX_METAL_CLANG_C,
+    SFX_METAL_CLANG_D,
+    // Stone Skull Hit
```

Comment:
Could we put comments about individual entries to the right? e.g.
```
SFX_METAL_CLANG_E, // Stone Skull Hit
```
This makes it so the vabid changes and info about the sounds are easier to read in my opinion

Reply:
Sure thing, one sec.